### PR TITLE
Adding a bash script to check for low-commitment fee anchor channels

### PR DESCRIPTION
**Pull request by TrezorHannes**
- sort your channels with low commit fee on top
- providing a list of channels which need your intervention (eg rake up fees, lower rebalancing)
- need review of umbrel's `lncli` feasibility (had some issues earlier, need to be tested)

### CODE REVIEW

File: anchor_check.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+#### GUIDE ###############################################
+# download this script with wget LINKTOFILE
+# assumes lncli is globally available: whereis lncli
+# execute with bash anchor_check.sh
```
Comment (blckbx) [inline anchor_check.sh:6]: Am I assuming correct that for Umbrel, you gotta start the script like so: `bash anchor_check.sh umbrel` ?

File: anchor_check.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+CHANNELS=$($anchor_list listchannels | jq '.channels | sort_by(.fee_per_kw | tonumber)')
+
+# Loop through each channel and extract the pubkey and fee_per_kw values
+while read -r line; do
+    # Extract the fee_per_kw and remote_pubkey values
+    FEE_PER_KW=$(echo "$line" | jq -r '.fee_per_kw')
+    PUBKEY=$(echo "$line" | jq -r '.remote_pubkey')
+
+    # Check if the pubkey is valid and get the alias name
+    ALIAS=$($anchor_list getnodeinfo --pub_key "$PUBKEY" | jq -r '.node.alias')
```
Comment (blckbx) [inline anchor_check.sh:36]:
```suggestion
    ALIAS=$($echo "$line" | jq -r '.peer_alias')
```
speed up that shit :rocket: 

before: real 0m23,405s
after: real 0m5,164s

**Review by blckbx — COMMENTED** [review]
:fire:  thanks for submission! 
one for clarification, one for :rocket:

File: anchor_check.sh (hunk 1 above)
Comment (TrezorHannes) [reply anchor_check.sh:6]: Yes, good point. I'll add this to the Guide section

**Review by blckbx — APPROVED** [review]

**Pull request merged**